reana-ui/src/pages/Workflows.js

```diff
@@ -30,7 +30,7 @@ export default class WorkflowsPage extends Component {
     return (
       <div>
         <Header />
-        <WorkflowList />
+        <WorkflowList token={this.props.token} />
```

Comment:
Is this way of passing parameters working? I am not getting any workflow in the list after logging in.

Reply:
Yes, it is the way of doing it. It is currently not working because we required to work with JWT in all the endpoints. So, the way I make it work is by locally changing WorkflowsList requests to access the `/workflows` endpoint in the old manner.